task/net-api/src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub enum NetError {
     Other = 5,
 }
 
-#[derive(Copy, Clone, Debug, Serialize, Deserialize)]
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Eq should be safe to implement on all of these too.

Reply:
Added in 0e319ce4